### PR TITLE
Use time zone names instead of time offsets

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

File: scripts/install/Install.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+namespace oat\taoDeliverySchedule\scripts\update;
+
+use oat\tao\model\ClientLibRegistry;
+
+/**
+ * 
+ * @author Aleh Hutnikau <hutnikau@1pt.com>
+ */
+$clientLibRegistry = ClientLibRegistry::getRegistry();
+$clientLibRegistry->register('moment', '../../../taoDeliverySchedule/views/js/lib/moment-with-locales.min');
```
Comment (ssipasseuth) [inline scripts/install/Install.php]: Do you absolutely need the alias moment for this to work ? If so please add a comment here so another won't be tempted to remove it.
However if it is not required, you can simply load it by requiring it as usual.
`define(['taoDeliverySchedule/lib/moment-with-locales.min'], function(moment){ ... });`

**Comment by ssipasseuth** [comment]
I have tested this along with https://github.com/oat-sa/tao-core/pull/573. Honestly, I am not sure I understand how this works. Upon save, I seem to have the same issue described here https://oat-sa.atlassian.net/browse/TAO-1545
- I created a delivery schedule
- edit the time
- changed the time zone and changed the time
- saved
- result -> the time has changed : is it expected ? Isn't it what this issue was about ?

**Comment by ssipasseuth** [comment]
depends on https://github.com/oat-sa/tao-core/pull/573

**Comment by hutnikau** [comment]
This is normal behavior. 
The problem was that after you open delivery and save it (without changing time and time zone) time was changed after saving.

**Pull request merged**